CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🐞 VAST no longer opens a random public port by default.
```

Comment:
This is a somewhat pathetic statement on its own. Could we add some context for the reader for why this happened in the past?

Reply:
Do you have a suggestion? Something like this maybe?

```
VAST no longer automatically accepts external connections when configured with broker.
```

Reply:
Here's a twist:

```
VAST no longer opens a random public port, which was used in the experimental VAST cluster mode in order to transparently establish a full mesh. 
```